waspc/data/Generator/templates/sdk/wasp/auth/useAuth.ts

```diff
@@ -1,6 +1,7 @@
 {{={= =}=}}
 import { deserialize as superjsonDeserialize } from 'superjson'
-import { useQuery, addMetadataToQuery } from 'wasp/client/operations'
+import { useQuery } from '../client/operations'
+import { addMetadataToQuery } from '../client/operations/queries/core'
```

Comment:
As explained in the issue's description, all the import paths inside the SDK have been converted from package imports to relative imports (also accounting for the updated paths).